pages/common/vifm.md

```diff
@@ -10,3 +10,23 @@
 - Open specified directories on left or right plane:
 
 `vifm {{path/to/directory_1}} {{path/to/directory_2}}`
+
+- Copy, paste, move and delete:
```

Comment:
```suggestion
- Copy, paste, move and delete:
```

Optional fix: CLI options can be demonstrated instead of file manager commands.
Note: TlDr pages describe CLI interface for commands most often. As such examples are still useful and there is no style guide requirement to describe just CLI the fix is optional.